core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -283,8 +290,7 @@ class WorkspaceService(
       }
       owners = ownersPolicy.map(policy => policy.memberEmails.map(_.value))
       canShare <- options.anyPresentFuture("canShare") {
-        if (accessLevel < WorkspaceAccessLevels.Read) Future.successful(false)
-        else if (accessLevel >= WorkspaceAccessLevels.Owner) Future.successful(true)
+        if (accessLevel >= WorkspaceAccessLevels.Owner) Future.successful(true)
```

Comment:
`No Access` is only the permission below reader. If the reader calls either `getWorkspace` or `getWorkspaceById` without at least read access, the entire endpoint will throw an exception.

Even if that were not the case, we'll always get the correct answer if we call Sam anyway.